database/schema.sql

```diff
@@ -26,7 +26,7 @@ CREATE TABLE reservations (
   month int NOT NULL,
   year int NOT NULL,
   time int NOT NULL,
-  numofpeopel int NOT NULL,
+  numofpeople int NOT NULL,
   restaurants_id int NOT NULL,
```

Comment:
Aren't you defining your schema in `database/index.js`? Is this file still necessary?

Reply:
I forgot to delete this file 👀